evalai/utils/challenges.py

```diff
@@ -160,3 +160,119 @@ def get_challenge_count(is_host=False, is_participant=False):
 
     for challenge in challenges:
         print_challenge_table(challenge)
+
+
+def print_phases(phases):
+    """
+    Prints all the phases of a particular challenge.
+    """
+    for phase in phases:
+        br = style("--------------------------------"
+                   "----------------------------------", bold=True)
+
+        phase_title = "\n{}".format(style(phase["name"], bold=True,
+                                          fg="green"))
+        challenge_id = "Challenge ID: {}".format(style(str(phase["challenge"]),
+                                                       bold=True, fg="blue"))
+        phase_id = "Phase ID: {}\n\n".format(style(str(phase["id"]),
+                                                   bold=True, fg="blue"))
+
+        title = "{} {} {}".format(phase_title, challenge_id, phase_id)
+
+        description = "{}\n\n".format(phase["description"])
```

Comment:
You want to parse the description with `bs4` (aka BeautifulSoup) here.
Similarly for `print_phase_details` and wherever applicable.